scripts/inner/kubestellar

```diff
@@ -161,16 +161,16 @@ if [ "$KUBECONFIG_DIR" == "" ]; then
     mkdir -p $KUBECONFIG_DIR
     export KUBECONFIG_DIR
 fi
-if [ "$SPACE_MANAGER_KUBECONFIG" == "" ]; then
-    SPACE_MANAGER_KUBECONFIG="${KUBECONFIG_DIR}/config"
-    export SPACE_MANAGER_KUBECONFIG
+if [ "$SM_CONFIG" == "" ]; then
+    SM_CONFIG="${KUBECONFIG_DIR}/config"
+    export SM_CONFIG
 fi
 if [ "$IN_CLUSTER" == "" ]; then
```

Comment:
Why is parameter conveyed via an environment variable rather than a command line flag?

It needs to be documented in `commands.md` in either case.